src/main/java/com/buergervereinHSH/BackendProject/auth/model/User.java

```diff
@@ -20,11 +20,14 @@
     private Set<Role> roles;
     private boolean enabled;
 
-    @JsonIgnore //brauchen wir das?
```

Comment:
@JsonIgnore ist da, um das Anzeigen von bestimmten Werten in dem verschickten json-Objekt zu vermeiden, wie Passwörter usw. Wenn du den Token per json nicht schickst, dann brauchts du das auch nicht :)

Reply:
I know! Hab auch nur den Kommentar rausgenommen, nicht das @JasonIgnore ;) Aber danke!